src/frontend/web/src/assets/styles/_colors.scss

```diff
@@ -0,0 +1,8 @@
+$grass-green: #86b362;
```

Comment:
Look at my PR :) We could then erase this file and change each place you used one of the colors to use `var(--<color-name>` instead.